ngraph/core/include/ngraph/op/deformable_convolution.hpp

```diff
@@ -160,6 +160,11 @@ namespace ngraph
 
                 void validate_and_infer_types() override;
 
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
Why do we need to have `evaluate` method for deformable convolution?

Reply:
Evaluate method: added temporarily, will be removed after merge of cpu implementation